services/aur/aur.service.js

```diff
@@ -75,9 +77,19 @@ class AurLicense extends BaseAurService {
     return { message: license, color: 'blue' }
   }
 
+  transform(json) {
+    const license = json.results.License
+    if (!license) {
+      throw new NotFound({ prettyMessage: 'license not found' })
```

Comment:
```suggestion
      throw new NotFound({ prettyMessage: 'not specified' })
```

What do you think about tweaking this?

Reply:
Agreed, that's better